staging/src/k8s.io/apiserver/pkg/admission/metrics/metrics.go

```diff
@@ -206,9 +206,9 @@ func (m *metricSet) reset() {
 
 // Observe records an observed admission event to all metrics in the metricSet.
 func (m *metricSet) observe(elapsed time.Duration, labels ...string) {
-	elapsedMicroseconds := float64(elapsed / time.Microsecond)
-	m.latencies.WithLabelValues(labels...).Observe(elapsedMicroseconds)
```

Comment:
Mainly, I can think of two ways this metric is (and can be) broken for people who have monitoring set up against these.  

The first group would be people who are using this metric **_and_** are aware that the metric is emitting in microseconds **_even though_** the label is in seconds. If they have set their alerts accordingly (this would be weird but not impossible), then we would break their monitoring with this fix. 

The second group are people who are using this metric **_as if_** this metric was working correctly, i.e. emitting latency in seconds. In that case, thresholds which are currently set for alerting would be off by orders of magnitude and this fix would actually make those alerts start working as intended.

Personally, I think we should just fix it.

Reply:
I'm also for changing as this is an actual bug, but can you add an item to the changelog that this is a change?

Reply:
This is part of the metrics overhaul planned for 1.14 where a number of metrics are changing and we're documenting every single case including what to change. As a middle ground, let's add an already deprecated metric that is called `admission_latencies_milliseconds_summary` so people who are affected by the break would only have to change the metric name and not do a unit conversion. I think this would work well, as 1.14 is the "metric migration" release, where we have deprecated metrics as well as the (new) best practice following metrics and the deprecated ones will be removed in 1.15.

This one is an interesting case as it's not just not following the best practice, but also incorrectly labels its unit. It will either way need a separate, additional changelog notice.

Reply:
agree with @brancz proposal. 
Added `admission_latencies_milliseconds` and `admission_latencies_milliseconds_summary` for Backward compatible. PTAL